src/Microsoft.Sbom.Api/Providers/ExternalDocumentReferenceProviders/CGExternalDocumentReferenceProvider.cs

```diff
@@ -74,7 +74,7 @@ protected override (ChannelReader<JsonDocWithSerializer> results, ChannelReader<
 
         protected override (ChannelReader<ScannedComponent> entities, ChannelReader<FileValidationResult> errors) GetSourceChannel()
         {
-            var (output, cdErrors) = sbomComponentsWalker.GetComponents(Configuration.BuildComponentPath?.Value);
+            var (output, cdErrors) = sbomComponentsWalker.GetComponents(Configuration.BuildComponentPath?.Value ?? Configuration.BuildDropPath?.Value);
```

Comment:
Considering this logic exists in a couple places, consider creating a method that just takes the `Configuration` and encapsulates this logic (i.e. the null-coalescing)